src/model/immutable/ContentState.js

```diff
@@ -116,13 +135,47 @@ class ContentState extends ContentStateRecord {
     );
   }
 
+  createEntity(
+    type: DraftEntityType,
+    mutability: DraftEntityMutability,
+    data?: Object
+  ): ContentState {
+    return createEntityInContentState(this, type, mutability, data);
+  }
+
+  mergeEntityData(
+    key: string,
+    toMerge: {[key: string]: any}
+  ): ContentState {
+    return updateEntityDataInContentState(this, key, toMerge, true);
```

Comment:
As you know, I don't love boolean arguments and return values. :) But I want to get this unblocked, so we can just revisit if needed.